e2e_tests/test_search.py

```diff
@@ -149,3 +176,87 @@ def test_click_on_clear_all_as_standard_user_redirects_to_browse_transferring_bo
         standard_user_page.get_by_role("link", name="Clear all").click()
 
         expect(standard_user_page).to_have_url(url)
+
+
+class TestOpenSearchResults:
```

Comment:
I think we shouldn't refer to opensearch. The e2e tests should be from the perspective of a user, who should not care about how we implement search under the hood.